csharp/README.md

```diff
@@ -2,10 +2,25 @@
 The C# driver is based on the cross-platform .NET 6 framework.
 
 ## Use TypeDB Driver for C#
-The driver will be distributed as a [Nuget](https://www.nuget.org/) package. More information will be provided in the next release!
+The driver is distributed as a series of [Nuget](https://www.nuget.org) packages. To use the driver, import the latest versions of [the driver](https://www.nuget.org/packages/TypeDB.Driver) and its [Pinvoke runtime](https://www.nuget.org/packages?q=TypeDB.Driver.Pinvoke) suitable for your platform. Here is an example from a `.csproj` for MacOS x86-64:
+```xml
+<PackageReference Include="TypeDB.Driver" Version={VERSION} />
+<PackageReference Include="TypeDB.Driver.Pinvoke.osx-x64" Version={VERSION} />
+```
+
+If you aim to build a platform-independent package, reference all the needed runtimes (it will affect the size of your application by downloading a respective set of platform-specific dynamic libraries):
+```xml
+<PackageReference Include="TypeDB.Driver.Pinvoke.osx-x64" Version={VERSION} />
+<PackageReference Include="TypeDB.Driver.Pinvoke.linux-x64" Version={VERSION} />
+<PackageReference Include="TypeDB.Driver.Pinvoke.win-x64" Version={VERSION} />
+<PackageReference Include="TypeDB.Driver.Pinvoke.osx-arm64" Version={VERSION} />
```

Comment:
Can you order these by `os-arch` please?
Also, we don't do linux-arm64?

Reply:
Okay, I'll add everything here. My initial idea was that we could forget to update `README` if we decide to add more platforms one day, so listing everything here is not necessary. At the same time, I wanted to show all the OS options and how they are spelled (`osx`, `linux`, `win`) with the way to spell archs (`x64` and `arm64`). That's why `linux-arm64` was redundant here =)